.travis.yml

```diff
@@ -1,7 +1,8 @@
 lannguage: ruby
 rvm:
+  - 2.1
   - 2.0.0
-  - 1.9.3
+  - rbx-2
```

Comment:
I _have_ seen it pass in our build (https://travis-ci.org/rabid/refinerycms-calendar/builds/57547713), so I believe it's an issue of timing (more specifically the generation of the dummy app taking too long on Rubinius). I don't think I can trigger a rebuild on the Refinery Travis, as I don't have the necessary permissions, but it may fix the build.

It could be removed, I had originally added in order to mirror the Rubies that RefineryCMS itself tests against. I think if it's possible to keep it in that would be best, to be consistent with RefineryCMS, but I can understand that if it's going to cause intermittent failures from being slow that might become frustrating.